terraform/workspace_variables/sandbox.tfvars

```diff
@@ -0,0 +1,19 @@
+# PaaS
+paas_sso_code          = ""
+paas_app_environment   = "sandbox"
+paas_web_app_instances = 2
+paas_web_app_memory    = 512
+
+#StatusCake
+statuscake_alerts = {
+  find-staging = {
+    website_name = "find-teacher-training-sandbox"
+    # to be changed to https://sandbox.find-postgraduate-teacher-training.service.gov.uk/ping once CDN is setup
```

Comment:
We could easily keep both actually. maybe `find-staging-paas` and `find-staging-cdn`